### DISCUSSION TITLE
Global case insensitive query setting

### DISCUSSION THREAD

**Discussion by moxplod** [alirezanet/Gridify#189] [Q&A] [answered]
Hi, I am starting to use this library extensively. Great library! Before this I was doing my own string parsing and expression creation. Now moving all that code to this library. 

Is there a way to globally make all the string filtering case insensitive? 

That way I do not have to tell my users to apply /i with each string parameter.

**Comment by alirezanet** [comment]
Hi @moxplod,
Unfortunately, there is no built-in configuration for global case sensitivity. However, you can append `/i` to your queries using a Custom GridifyQuery class or some middleware in your application. Please feel free to open a feature request issue if you believe that having such a configuration would be beneficial. We can then investigate it further.

**Comment by moxplod** [comment, marked as answer]
Comlpeted https://github.com/alirezanet/Gridify/pull/194
